types/mocha.d.ts

```diff
@@ -0,0 +1,214 @@
+// Type definitions for mocha 2.2.5
```

Comment:
I will keep the manual copies of typescript definitions from definitely typed in our repo, but renamed the directory to /typings so it aligns with TSD if we change our minds later. I think Bolt doesn't have enough dependancies to justify TSD quite yet.